content/docs/guides/crossguard/configuration.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Configurable Policy Packs
+meta_desc: Configuration allows you to write flexible policies that can be reused across you organization.
+linktitle: Configuration
+weight: 2
+
+menu:
+  userguides:
+    parent: crossguard
+---
+<!-- markdownlint-disable ul code -->
+{{% crossguard-preview %}}
+
+## Overview
+
+Configuration allows you to author flexible Policy Packs that can be reused across your entire organization. As an organization administrator, you can use a single Policy Pack yet vary configuration (e.g. enforcement level, the allowed instance types, cost allowances, etc.) from one Policy Group to the next. For example, you may have a Policy Group for your non-production stacks that allow smaller instance types, while your Production Policy Group allows for use of large instance types.
+
+Configuration schema defined using per policy, and then the actual configuration can be set via a form in the Pulumi Console or using a JSON file.
```

Comment:
Configuration schema "is defined per policy, and then the actual....."  ?

Reply:
... what is the question?

Reply:
oh i see.. will fix!